homestead.root/usr/share/clearwater/infrastructure/scripts/homestead

```diff
@@ -22,6 +22,12 @@ then
     hs_listen_port=$hss_port
 fi
 
+# If homestead_tw_timer is not set, default to the minimum of 6s
+if [[ -z "$homestead_tw_timer" ]]
+then
+  homestead_tw_timer=6
```

Comment:
I'm not sure I like the name "*_tw_timer". That won't mean anything to most users I suspect. I would suggest "homestead_diameter_watchdog_timer"